### PR TITLE
Fixing typo errors in docs README file

### PR DESCRIPTION
**Pull request by DukeDavis12**
Fixing typo errors in docs README file. Includes
   spelling errors as well as grammatical errors.

Signed-off-by: Davis Benny <davis.benny@intel.com>

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -41,15 +41,15 @@ Details of the Secure Device Onboard protocol can be found in the Secure Device
 
 The Secure Device Onboard project provides software that helps simplify the implementation of an end-to-end Secure Device Onboard capability.  The following components are available:
 
-**Client SDK** - The Client SDK provides devices with an implementation of the DI protocol, which initializes the device with credentials, the TO1 protocol, which connects the device to a Rendezvous Service, and the TO2 Protocol, which connects the device to an IOT Platform Service.  The Client SDK supports Linux on X86 processors, as well as Linus and SELinux on ARM processors.
```
Comment (DukeDavis12) [inline docs/index.md:44]: Linus --> spelling error.

File: docs/index.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
+Today, organizations that deploy IoT devices are struggling with manual methods to securely connect devices with IoT platforms. Traditional onboarding processes are time-intensive and require IT expertise. The Secure Device Onboard solution simplifies this process for end customers while simultaneously establishing trust across all organizations who manufacture, distribute, or sell the device and associated IoT solutions.
 
-During deployment of a device enable with the Secure Device Onboard solution, all the installer needs to do is add power to the device and connect it to the Internet. The device automatically and rapidly connects to the IoT platform in a secure and trusted manner. Secure Device Onboard combines embedded hardware security credentials with a chain-of-ownership security credential and Rendezvous service to secure the onboarding process and broker the connection to the authorized device management service provider. Today Secure Device Onboard supports Intel<sup>®</sup> Enhanced Privacy ID (Intel<sup>®</sup> EPID) and Elliptic Curve Digital Signature Algorithm-based embedded hardware security credentials.
```
Comment (DukeDavis12) [inline docs/index.md:7]: During deployment of a device enable with the Secure Device Onboard solution -->  During deployment of a device enabled with the Secure Device Onboard solution.

File: docs/index.md (hunk 1 above)
Comment (DukeDavis12) [inline docs/index.md:44]: Linus --> spelling

File: docs/index.md [hunk 3]
```diff
@@ -41,15 +41,15 @@ Details of the Secure Device Onboard protocol can be found in the Secure Device
...
+**Client SDK** - The Client SDK provides devices with an implementation of the DI protocol, which initializes the device with credentials, the TO1 protocol, which connects the device to a Rendezvous Service, and the TO2 Protocol, which connects the device to an IOT Platform Service.  The Client SDK supports Linux on X86 processors, as well as Linux and SELinux on ARM processors.
 
 **Manufacturing Toolkit** - The Manufacturing Toolkit software enables the manufacturer to load credential into devices.  It implements the DI protocol, and includes support for database storage of credentials and is designed to integrate into the manufacturers IT processes.
 
 **Reseller Toolkit** - The Reseller Toolkit enables distributors and System Integrators to manage the ownership credential over the devices lifetime.  It implements the DI protocol.
 
-**Rendezvous Service** - The Rendezvous Service software provides a complete Rendezvous service, and implements the TO0 and TO1 protocols required for device registration and discovery.  It can be run in the cloud or on-premise and includes an interface to support key attestation.
```
Comment (DukeDavis12) [inline docs/index.md:50]: unnecessary comma

File: docs/index.md [hunk 4]
```diff
@@ -41,15 +41,15 @@ Details of the Secure Device Onboard protocol can be found in the Secure Device
...
+**Rendezvous Service** - The Rendezvous Service software provides a complete Rendezvous service and implements the TO0 and TO1 protocols required for device registration and discovery.  It can be run in the cloud or on-premise and includes an interface to support key attestation.
 
-**IOT Platform SDK** - The IOT Platform SDK provides micro-services to facilitate integration of an IOT Platform in the Secure Device Onboard processes. It implements the TOO protocol required for device registration, and the TO2 protocol required for device ownership transfer.
```
Comment (DukeDavis12) [inline docs/index.md:52]: the noun phrase "Integration" seems to be missing a determiner before it. So adding an article.

File: docs/index.md [hunk 5]
```diff
@@ -58,19 +58,19 @@ A directory of documentation and source repositories for the Secure Device Onboa
 
 ## The Secure Device Onboard Process
 
-This section describes a typical process in which Secure Device Onboard is used to automatically and securely onboard a device onto an owners network.  Those process include:
```
Comment (DukeDavis12) [inline docs/index.md:61]: the plural demonstrative  "those"  is modifying the singular noun.  So changing process to processes.

File: docs/index.md [hunk 6]
```diff
@@ -58,19 +58,19 @@ A directory of documentation and source repositories for the Secure Device Onboa
...
+**Credential Management** – The manufacturers and supply chain integrators run a tool to manage security keys and ownership credentials that correspond to each device. These credentials do not reside on the device and are used to negotiate trust as devices are bought and sold.
 
 **Transfer To IoT Platform** – The ownership credentials are imported into an SDO-enabled IoT Platform. 
 
-**Device Registration** – The ownership credentials are synchronized between the device management service and the rendezvous server.   
+**Device Registration** – The ownership credentials are synchronized between the device management service and the rendezvous server.
 
 **Onboarding** - The process of verifying the device identity with the rendezvous server and brokering a trusted connection to the intended IoT platform when a device is powered on and connected to the Internet. 
 
-The following diagram shows an example of a possible IoT supply chain and the steps taken by each participant in the Secure Device Onboard service. Each step is described in more detail in following sections.
```
Comment (DukeDavis12) [inline docs/index.md:73]: missing article.

File: docs/index.md [hunk 7]
```diff
@@ -113,7 +113,7 @@ The system integrator acquires the IoT Platform provider’s public key and uses
 
 The IoT Platform provider registers the ownership voucher with the *Rendezvous Service*. The *Rendezvous Service* may be operated by a cloud service provider, the IOT Platform service provider, or other service provider entity. Behind the scenes, the IOT Platform uses the Transfer Ownership 0 (TO0) protocol to communicate with the Rendezvous server. 
 
-To facilitate easy integration with Secure Device Onboard, the IOT Platform can use the Secure Device Onboard *IOT Platform SDK*, which implements the protocol features required by the IPT Platform to communicate with the Rendezvous service, and provides an easy REST-based interface to communicate between the SDK components and the IOT Platform.
```
Comment (DukeDavis12) [inline docs/index.md:116]: IPT -> spelling error.

File: docs/index.md [hunk 8]
```diff
@@ -154,9 +154,9 @@ The *All-In-One Demo* runs the Rendezvous Service, a simulated IOT Platform usin
 
 Manufacturers interested in implementing support for Secure Device Onboard in their factories can learn more by referring to the Secure Device Onboard *Manufacturer Enablement Guide*.
 
-### Distributers and Resellers
```
Comment (DukeDavis12) [inline docs/index.md:157]: Wrong Usage -> Distributers

File: docs/community.md [hunk 9]
```diff
@@ -3,7 +3,7 @@ Welcome to the Secure Device Onboard community!
 
 ## Join the Discussion
 
-Report issues using each component's respective github repository.
+Report issues using each component's respective Github repository.
```
Comment (trbehera) [inline docs/community.md]: GitHub => GitHub

That is a more standard way to refer.

File: docs/community.md [hunk 10]
```diff
@@ -59,14 +59,14 @@ Your commit email address must match your GitHub email address. For more informa
...
-A pull request cannot merged until it has passed these status checks:
+A pull request cannot be merged until it has passed these status checks:
 
 - The build must pass on Jenkins
 - The pull request must be approved by at least two reviewers without any outstanding requests for changes
 
 ## Reporting Issues
 
-Report issues using each component's respective github repository.
+Report issues using each component's respective Github repository.
```
Comment (trbehera) [inline docs/community.md]: Github => GitHub

**Review by trbehera — CHANGES_REQUESTED** [review]

**Review by trbehera — DISMISSED** [review]

**Review by KiranSukhavasi — DISMISSED** [review]

**Review by v-vasavi — COMMENTED** [review]
1. all the installer needs to do is "to"  add power would be good to add a "to" there
2. Line 61 - Should it be "These processes" instead of Those

**Comment by trbehera** [comment]
> all the installer needs to do is "to"  add power would be good to add a "to" there

Updated.

> Line 61 - Should it be "These processes" instead of Those

Line 61=> It feels okay to use 'Those' here.

**Review by trbehera — APPROVED** [review]

**Review by KiranSukhavasi — APPROVED** [review]

**Pull request merged**